homework/test/test-list.js

```diff
@@ -5,4 +5,9 @@ describe('Test > ', function() {
   it('will test some pure functions', function() {
     // Write any pure function assertion here.
   });
+  
+  it('simple test case', function(){
+	  assert.equal(1+1,2);
```

Comment:
Please put this expression in one method of one manager, and call it here rather than just put the expression here.